### PR TITLE
Adds SPEC file for the plugin RPM package

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes: #7

### CODE REVIEW

**Comment by navidshaikh** [comment]
@strzibny: can you please review this?

**Comment by strzibny** [comment]
@navidshaikh 

From what I can see:
- Requires(pre): shadow-utils is not needed here

Btw do you plan on submitting a proper review request to Fedora? I could review there...

File: vagrant-adbinfo.spec [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Source0: https://rubygems.org/gems/%{vagrant_plugin_name}-%{version}.gem
+Requires(posttrans): vagrant
+Requires(preun): vagrant
+Requires: vagrant
+BuildRequires: ruby(release)
+BuildRequires: rubygems-devel >= 1.3.6
+BuildRequires: ruby 
+BuildRequires: vagrant
+BuildArch: noarch
+Provides: vagrant(%{vagrant_plugin_name}) = %{version}
```
Comment (navidshaikh) [inline vagrant-adbinfo.spec:20]: @strzibny : removed the `shadow-utils` from `Requires`

**Comment by navidshaikh** [comment]
@strzibny : Yes, we (Bex or I) plan for a proper review request at Fedora.

**Comment by strzibny** [comment]
@navidshaikh looks alright, if you submit it to Fedora I will build and run it.

File: vagrant-adbinfo.spec [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+%files doc
+%doc %{vagrant_plugin_docdir}
+%{vagrant_plugin_instdir}/Gemfile
+%doc %{vagrant_plugin_instdir}/README.md
+%{vagrant_plugin_instdir}/Rakefile
+%{vagrant_plugin_instdir}/Vagrantfile
+%{vagrant_plugin_instdir}/CONTRIBUTING.md
+%{vagrant_plugin_instdir}/LICENSE
+%{vagrant_plugin_instdir}/MAINTAINERS
+%{vagrant_plugin_instdir}/vagrant-adbinfo.gemspec
```
Comment (navidshaikh) [inline vagrant-adbinfo.spec:83]: Packaged CONTRIBUTING.md, LICENSE and MAINTAINERS

**Comment by navidshaikh** [comment]
@strzibny : Can you please review the updated SPEC file, I think we will get it merged, I have to keep on rebasing the PR.

**Comment by strzibny** [comment]
I also built it and tested it on Fedora 22. Looks good to be merged.

**Comment by bexelbie** [comment]
LGTM

**Pull request merged**